common/lib/xmodule/xmodule/contentstore/mongo.py

```diff
@@ -1,52 +1,49 @@
+"""
+MongoDB/GridFS-level code for the contentstore.
+"""
+import os
+import json
 import pymongo
 import gridfs
 from gridfs.errors import NoFile
-
-from xmodule.contentstore.content import XASSET_LOCATION_TAG
-
-import logging
-
-from .content import StaticContent, ContentStore, StaticContentStream
-from xmodule.exceptions import NotFoundError
 from fs.osfs import OSFS
-import os
-import json
 from bson.son import SON
+
+from mongodb_proxy import autoretry_read
 from opaque_keys.edx.keys import AssetKey
+from xmodule.contentstore.content import XASSET_LOCATION_TAG
+from xmodule.exceptions import NotFoundError
 from xmodule.modulestore.django import ASSET_IGNORE_REGEX
 from xmodule.util.misc import escape_invalid_characters
+from xmodule.mongo_connection import connect_to_mongodb
+from .content import StaticContent, ContentStore, StaticContentStream
 
 
 class MongoContentStore(ContentStore):
-
+    """
+    MongoDB-backed ContentStore.
+    """
     # pylint: disable=unused-argument
-    def __init__(self, host, db, port=27017, user=None, password=None, bucket='fs', collection=None, **kwargs):
+    def __init__(self, host, db,
+                 port=27017, tz_aware=True, user=None, password=None, bucket='fs', collection=None, **kwargs
+                 ):  # pylint: disable=bad-continuation
```

Comment:
That's the style I prefer as well - and PEP8 likes it. But pylint still flags it. I'll change it to that style and keep the pylint disable.